docs/headerMiddleware.md

```diff
@@ -0,0 +1,175 @@
+# gpii.express.middleware.headerSetter
+
+This component sets one or more [HTTP response headers](https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html#sec6.2)
+based on the contents of `options.headers` (see below).
+
+## Using this grade
+
+This grade can only do its job when it is added as a child component of either a `gpii.express` or `gpii.express.router`
+instance.  Here's an example of adding a single header to all responses send from a `gpii.express` instance:
+
+    fluid.defaults("my.express.grade", {
+        gradeNames: ["gpii.express"],
+        path: 8080,
+        components: {
+            headerSetter: {
+                type: "gpii.express.middleware.headerSetter",
+                options: {
+                    headers: {
+                        server: {
+                            fieldName: "Server",
+                            template:  "Custom gpii-express Server/1.0.0"
+                        }
+                    }
+                }
+            }
+            // TODO:  You, the reader, should add at least one middleware component here that will respond to the user.
```

Comment:
No.  Our handlers do not yet work like those in kettle.  They require an enclosing piece of middleware, one that has `gpii.express.handlerDispatcher` mixed into its `gradeNames`.